packages/shared-components/src/components/onto-footer/onto-footer.tsx

```diff
@@ -1,23 +1,57 @@
-import { Component, Host, h } from '@stencil/core';
+import { Component, Host, h, State } from '@stencil/core';
+import { LicenseContextService, ProductInfo, ServiceProvider, SubscriptionList } from '@ontotext/workbench-api';
+import { TranslationService } from '../../services/translation.service';
 
+/**
+ * OntoFooter component for rendering the footer of the application.
+ * This component displays information about GraphDB, RDF4J, Connectors, and Workbench versions,
+ * as well as copyright information.
+ */
 @Component({
   tag: 'onto-footer',
   styleUrl: 'onto-footer.scss',
   shadow: false,
 })
 export class OntoFooter {
+  /** State variable to store product information */
+  @State() private productInfo: ProductInfo;
```

Comment:
I don't think we need a state for this component. This data is loaded just once on app bootstrap and  is not a subject of a runtime change afterwards, so we can keep it safely as a normal field.